ui-v2/app/templates/dc/services/show/upstreams.hbs

```diff
@@ -6,11 +6,19 @@
         Upstreams are services that may receive traffic from this gateway. Learn more about configuring gateways in our
         <a href="{{env 'CONSUL_DOCS_URL'}}/connect/ingress_gateway.html" target="_blank" rel="noopener noreferrer">documentation.</a>
       </p>
+    {{#let item.Service.Namespace as |nspace|}}
```

Comment:
This doesn't matter at all, but just so you know, most 'page' templates have a `nspace` variable present, which is the name of the current namespace. This is perfectly fine though I think, so just a FYI.